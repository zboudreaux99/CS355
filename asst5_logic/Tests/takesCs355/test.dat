takes(S, 'cs355'), write(S), nl,fail.
