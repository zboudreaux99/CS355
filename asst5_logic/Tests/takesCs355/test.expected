Allison, Chen-chang
Angeles, Eric
Balingit, Catherine
Bannon, Carlin
Barnes, Maria
Bryant, Natasha
Cai, Candaice
Candies, Iman
Corton, Lawanda
Dammalapati, Tameishaon
Fei, Tawana
Gary, Anthony
Hoffmann, Rachael
Irintchev, Adel
Ishfaq, Abdullah
Kidd, Ahmad
Legg, Lakesha
Li, Lawson
Littrell, Teresa
Mack, Lakesha
Mason, Jenifer
Mccormack, Christin
Mcghee, Valentina
Miller, Crystal
Mills, Jinghua
Minor, Glenn
Mitchell, Jeremiah
Poe, Angela
Porter, Aretha
Raines, Corrigan
Reasor, Hamid
Reyes, Gustavo
Rocci, Gustavo
Rollins, Amy
Rusak, Ricardo
Saddi, Deveda
Sause, Rajiv
Settar, Stacie
Shi, Nichelle
Shuniak, Tameishaon
Sullivan, Dennison
Ward, Alisa
Yoder, Gustavo
no
